src/containers/ChainButton/ChainButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Button as ButtonComponent } from 'decentraland-ui'
+import { Props } from './ChainButton.types'
+import ChainCheck from '../ChainCheck'
+
+const disabledStyle = { opacity: 0.5, cursor: 'not-allowed', transform: 'none' }
+
+export default class ChainButton extends React.PureComponent<Props> {
```

Comment:
I stuck to the class since all the components in this repo are class components tbh, since that's the approach we are taking with the other repos (marketplace, builder, account).